### ISSUE TITLE
[Pods 2.8 Bug] Order relationships by dragging

### ISSUE THREAD

**Issue by JoryHogeveen** [pods-framework/pods#6094]
## Pods 2.8 version

2.8.0-b-5

## Description

In Pods 2.7 you could order relationships by dragging. Since 2.8 you only have the arrows. This is not a very convenient way to order if you have more than 2-3 relationships.

**Comment by sc0ttkclark** [comment]
@zrothauser What was the limitation here? Can we allow both?

**Comment by zrothauser** [comment]
@sc0ttkclark We talked about that I was running into some problems with the drag-and-drop library and it was taking a while, moved this to the 2.9 list to save on time.

**Comment by sc0ttkclark** [comment]
Okay great, let's revisit after simple repeatable fields then

**Comment by JoryHogeveen** [comment]
Is it correct that the file field still uses the "older" DFV code? I see a jQuery.sortable() reference there:
https://github.com/pods-framework/pods/blob/release/2.8/ui/js/dfv/src/fields/file/views/file-upload-list.js#L62-L68

**Comment by zrothauser** [comment]
@JoryHogeveen The file field uses the old code (with a wrapper around it, to try to force it to work inside React), but the relationship field does not.

**Comment by JoryHogeveen** [comment]
Another user mentioned this issue: https://wordpress.org/support/topic/multiple-select-list-view-drag-and-drop/

**Comment by sc0ttkclark** [comment]
Fixed via https://github.com/pods-framework/pods/pull/6390

**Comment by kemalkerem** [comment]
Hello everyone, I'm waiting for this version to be released. I installed version 2.8.9-a-1 to check. When I checked, I saw that it had some problems. The problems are as follows;

The dragged element on the listing page can only be dragged once. After being dragged and dropped, it doesn't drag anymore.
If there are too many elements on the listing screen (for example, 30 items), the position of the item may shift when dragging from the bottom to the top. The dragged item and the mouse cursor do not move together.
